doc/user/content/concepts/indexes.md

```diff
@@ -82,27 +96,28 @@ than from storage.
 For best practices on using indexes, and understanding when to use indexed views
 vs. materialized views, see [Usage patterns](#usage-patterns).
 
-```mzsql
-CREATE INDEX idx_on_my_mat_view ON my_mat_view_name(...) ;
-```
-
 ## Indexes and clusters
 
-Indexes are local to a cluster. Queries in a different cluster cannot use the
+Indexes are local to a cluster. Queries from a different cluster cannot use the
 indexes in another cluster.
 
-For example, to create an index in the current cluster:
+For example, the following operation creates an index in the current cluster:
 
 ```mzsql
 CREATE INDEX idx_on_my_view ON my_view_name(...) ;
 ```
 
-You can also explicitly specify the cluster:
+Only queries in the current cluster can use the index.
+
+Similarly, the following operation creates an index in the specified cluster
+named `active_cluster`:
 
 ```mzsql
 CREATE INDEX idx_on_my_view IN CLUSTER active_cluster ON my_view (...);
 ```
 
+Only queries in the `active_cluster` cluster can use the index.
```

Comment:
Might be worth additionally pointing out that queries in the cluster _might_ not use the indexes, too. This depends on the query patterns. We could point to the [troubleshooting section for transformations](https://materialize.com/docs/transform-data/troubleshooting/#why-is-my-query-slow)?

Reply:
👍 will do

Reply:
Linked to the transformation optimization page since the troubleshooting page just mentions the keys ... and not all the other ways.  The transformation optimization page will also be updated as a separate PR as part of action items from the index101 meeting).